paths/accounts/search.yaml

```diff
@@ -0,0 +1,76 @@
+---
+summary: Search across projects
+description: Search for keys and translations in all account projects
+operationId: account/search
+tags:
+- Accounts
```

Comment:
I think this will put `Search across projects` under `Accounts` in the sidebar?
I am not sure if this is where we should put it. To me it is probably even a new sidebar header "Search".
The result of an endpoint should reflect some resource of the header, so to me everything below `Accounts` should return some sort of account (short version, detail, list, ...) or have mutations on accounts.